### ISSUE TITLE
scale_massflows fails  `techflows=true`

### ISSUE THREAD

**Issue by spuhlerd** [santiago-sanitation-systems/Santiago.jl#71]
IN README:

## 4) scale massflows for 100 people

fewSys = scale_massflows.(fewSys, 100)

AND: runfile including links to download inputs sent by mail

**Comment by scheidan** [comment]
Your example (in the email)
```Julia
scale_massflows!(allSys, 10) 
``` 
doesnt work because `allSys` is a vector. You need to add a `.` for broadcasting (a standard Julia feature):
```Julia
scale_massflows!.(allSys, 10) 
``` 

`scale_massflows` and `scale_massflows!` take a single system as arguments, this is similar to:
```Julia
sysappscore!(sys)
ntechs!(sys)
nconnections!(sys)
connectivity!(sys)
template!(sys)
```
If you want to apply it to a vector fo systems, you must use the dot for broadcasting as in the examples in the readme.

**Comment by scheidan** [comment]
If massflows are computed _with the argument `techflows=true`_,
`scale_massflows` fails with:
```Julia
ERROR: ArgumentError: broadcasting over dictionaries and `NamedTuple`s is reserved
Stacktrace:
 [1] broadcastable(#unused#::Dict{String, NamedArrays.NamedArray})
   @ Base.Broadcast ./broadcast.jl:683
 [2] broadcasted(::Function, ::Dict{String, NamedArrays.NamedArray}, ::Int64)
   @ Base.Broadcast ./broadcast.jl:1312
 [3] scale_massflows!(sys::System, n_users::Int64)
   @ Santiago ~/Santiago.jl/src/massflow.jl:469
 [4] scale_massflows(sys::System, n_users::Int64)
   @ Santiago ~/Santiago.jl/src/massflow.jl:480
 [5] top-level scope
   @ REPL[25]:1
```

**Comment by scheidan** [comment]
fixed by fac6eb2
